### DISCUSSION TITLE
How to update Nuclei engine

### DISCUSSION THREAD

**Discussion by thvinha17010** [projectdiscovery/nuclei#1967] [Q&A] [answered]
How to update Nuclei engine 2.6.9 to 2.7.0
I use command  `go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest`
but it not update

**Comment by forgedhallpass** [comment, marked as answer]
Please refer to this discussion: https://github.com/projectdiscovery/nuclei/discussions/1723#discussioncomment-2374687

**Reply by forgedhallpass** [reply]
![image](https://user-images.githubusercontent.com/13679401/167405539-c80a0186-f57f-4c0a-bfb5-fb57d5e5d194.png)
